.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+## Scope 
+Description of feature / issue, and links to issue...
```

Comment:
I don't think we need to have these descriptions in each file. I feel the point of the templates is to make it quick and easy to write up the Issue/Pull Request, having this in here means the developer needs to select it and remove it each time. Small I know.

Opinions?